### PR TITLE
Update rendering.js

### PR DESCRIPTION
**Pull request by Cobuch**
revised code version

### CODE REVIEW

**Comment by Cobuch** [comment]
I also changed the font color for the score

File: scripts/rendering.js [hunk 1]
```diff
@@ -48,9 +48,16 @@ const render = (function(){
 		context.fillRect(mnBarPos.x + mnBordW, mnBarPos.y + mnBordW,
 			mnWidth, mnBarConf.height - 2 * mnBordW)
 	}
+	background = new Image();  // "Создаём" изображение
+       	background.src = 'pictures/Full-Background.png';  // Источник изображения
+      	function drawBackground(context) {  // Событие onLoad, ждём момента пока загрузится изображение
+       		context.drawImage( background, 0, 0);  // Рисуем изображение от точки с координатами 0, 0
```
Comment (MatvienkoDO) [inline scripts/rendering.js:54]: Usage of extended version of that function

Comment (MatvienkoDO) [reply scripts/rendering.js:54]: Created issue

File: scripts/rendering.js [hunk 2]
```diff
@@ -48,9 +48,16 @@ const render = (function(){
...
 	}
+	background = new Image();  // "Создаём" изображение
+       	background.src = 'pictures/Full-Background.png';  // Источник изображения
+      	function drawBackground(context) {  // Событие onLoad, ждём момента пока загрузится изображение
+       		context.drawImage( background, 0, 0);  // Рисуем изображение от точки с координатами 0, 0
+      	}
+
 
 	return function(diff, context) {
 		clearAll(context)
```
Comment (MatvienkoDO) [inline scripts/rendering.js:59]: Try to remove it if background is drawn on whole canvas

Comment (MatvienkoDO) [reply scripts/rendering.js:59]: created issue

File: scripts/rendering.js (hunk 1 above)
Comment (MatvienkoDO) [reply scripts/rendering.js:54]: deferred

File: scripts/rendering.js (hunk 2 above)
Comment (MatvienkoDO) [reply scripts/rendering.js:59]: deferred

**Pull request merged**